docs/src/docs/asciidoc/user/serializers-copiers.adoc

```diff
@@ -109,7 +109,7 @@ Implement the following interface, from package `org.ehcache.spi.serialization`:
 
 [source,java,indent=0]
 ----
-include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=21..-1]
+include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=20..-1]
```

Comment:
It's an API file... not going to change any time soon.  To me there are two better fixes:

1. Write a Java syntax aware IncludeProcessor for AsciidoctorJ
2. Get rid of the include entirely (I think it's overkill).